README.md

```diff
@@ -43,44 +60,80 @@ cargo install --path helix-term
 
 This will install the `hx` binary to `$HOME/.cargo/bin`.
 
-Helix also needs its runtime files so make sure to copy/symlink the `runtime/` directory into the
-config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overriden
-via the `HELIX_RUNTIME` environment variable.
+#### Runtime
 
-Packages already solve this for you by wrapping the `hx` binary with a wrapper
-that sets the variable to the install dir.
+> Note: Installing runtime is not required for linux packages or binaries.
 
-> NOTE: running via cargo also doesn't require setting explicit `HELIX_RUNTIME` path, it will automatically
-> detect the `runtime` directory in the project root.
+Helix requires extra files for syntax highlighting and themes. To install, copy the `runtime/` folder to the directory bellow.
```

Comment:
```suggestion
Helix requires extra files for syntax highlighting and themes. To install, copy the `runtime/` folder to the directory below.
```

Reply:
Let's recommend symlinking, or setting the HELIX_RUNTIME variable. Copying works but it's a brute force method that will cause issues in the future.